Processing/Explore_New_Parks.Rmd

```diff
@@ -0,0 +1,303 @@
+---
+title: "Data_Availability_Parks"
+author: "KEC"
+date: "2024-11-05"
+output: word_document
+editor_options: 
+  chunk_output_type: inline
+---
+
+Define park units that you want to explore data for
+
+```{r}
+
+parks <- c("VALL","CHIR", "FOBO", "CORO", "CACH", "HUTR", "NAVA", "ZION", "CEBR")
+```
+
+
+```{r setup, include=FALSE}
+source("setup.R")
+
+pal <- c("black",
```

Comment:
I made this edit in the deep dive .Rmd, but just a little trick, I like to make these palettes a list instead of a vector because then you can type `pal$` and all your color options will pop up, so you don't have to remember the index numbers.

Reply:
Updated to use palette.